### ISSUE TITLE
Add Babelfishfile

### ISSUE THREAD

**Issue by vmarkovtsev** [bblfsh/sdk#317]
We hit this recurring problem from time to time: drivers update, some problems are fixed and thus some of our workarounds crash. There can be new bugs added as well - e.g. when a native parser is changed or the SDK is refactored. So we propose to add `Babelfishfile` which pins the driver versions similar to `package.json` or `requirements.txt`. We expect it to work like

```
bbfshctl driver install -f /path/to/Babelfishfile
```

or

```
cat Babelfishfile | docker exec bblfshd driver install -f -
```

**Comment by juanjux** [comment]
Good idea.

**Comment by eiso** [comment]
I like this idea!

**Comment by dennwc** [comment]
@mcuadros Can you please move this issue to [bblfshd](https://github.com/bblfsh/bblfshd)?

**Comment by bzz** [comment]
We already have an issue in bblfshd - may be we could re-purpose it/merge this with https://github.com/bblfsh/bblfshd/issues/181 ?

**Comment by vmarkovtsev** [comment]
@bzz But the issue you point to is quite different: it is proposed to load drivers from file system there.

**Comment by dennwc** [comment]
Yes, I think it should be a separate issue. Waiting for it to be transferred to `bblfshd`.
